legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/DDL/toDDL.pure

```diff
@@ -92,7 +92,7 @@ Class meta::relational::functions::database::Res
 }
 
 function meta::relational::functions::database::loadCsvDataToDbTable(csvData:List<String>[*],
```

Comment:
do we have tests for this function anywhere? we should add a test for having more than one db in this function.